go.mod

```diff
@@ -11,18 +11,17 @@ require (
 	github.com/go-openapi/jsonreference v0.19.3
 	github.com/go-openapi/spec v0.19.3
 	github.com/go-openapi/swag v0.19.5
-	github.com/golang/protobuf v1.3.3
-	github.com/google/go-cmp v0.4.0 // indirect
+	github.com/golang/protobuf v1.4.2
```

Comment:
we should not bump dependencies this much on release branches... backporting the lazy loading is fine, but have it lazily do whatever was being done on that release branch (for 1.21 and 1.20, `json marshal → json unmarshal → map[string]interface{} → proto`) rather than bumping dependencies significantly

Reply:
@muvaf Are you able to undo that specific change?

Reply:
might be easier to revert both commits, repick just the `Add lazy marshaling to handler.go` commit, and resolve the conflict in the lazy proto marshaler to do the `json marshal → json unmarshal → map[string]interface{} → proto` bit

Reply:
Thanks for the feedback, Yes, I can do that for both 1.20 and 1.21 branches in the next hour and let you know.

Reply:
@apelisse The PRs are ready:
* release-1.20: https://github.com/kubernetes/kube-openapi/pull/268
* release-1.21: https://github.com/kubernetes/kube-openapi/pull/267